pkg/minikube/storageclass/storageclass_test.go

```diff
@@ -0,0 +1,259 @@
+package storageclass
```

Comment:
Need a license. Run `./hack/boilerplate/fix.sh` to add it. This will also fix jenkins.

Reply:
Thanks mate! Fixed!